stdlib/@python2/copyreg.pyi

```diff
@@ -3,7 +3,7 @@ from typing import Any, Callable, Hashable, Optional, SupportsInt, TypeVar, Unio
 _TypeT = TypeVar("_TypeT", bound=type)
 _Reduce = Union[tuple[Callable[..., _TypeT], tuple[Any, ...]], tuple[Callable[..., _TypeT], tuple[Any, ...], Optional[Any]]]
 
-__all__: list[str]
+__all__ = ["pickle", "constructor", "add_extension", "remove_extension", "clear_extension_cache"]
```

Comment:
Seems like this file doesn't actually exist

Reply:
Indeed...